### ISSUE TITLE
[FEATURE] RTNeural mod loader in BYOD

### ISSUE THREAD

**Issue by RustoMCSpit** [Chowdhury-DSP/BYOD#363]
**Describe the solution you'd like**
using BYOD as a means to make a GUI, context explained better with link

**Additional context**
https://github.com/brummer10/Ratatouille.lv2/issues/12

**Comment by jatinchowdhury18** [comment]
Yeah, it could be cool to have a "Ratatouille" module in BYOD, although I would need to look a little bit closer at the source code.

I'll add it to my list :).

**Comment by RustoMCSpit** [comment]
if you do, have a way to separate irs and amps too. with the addition of this youve essentially got a less fancy looking amplitube 5

![image](https://github.com/Chowdhury-DSP/BYOD/assets/134429563/e870810c-3b26-45b1-988e-8afbadbf41fc)

**Comment by RustoMCSpit** [comment]
and if this goes anywhere youd actually be insanely close to an amplitube remake https://github.com/sdatkinson/NeuralAmpModelerPlugin/issues/415

**Comment by RustoMCSpit** [comment]
wait shit are there any drag and drop pedal ui makers? sorry i know this is going off topic but if there was a way to automate the process... whatever just a silly idea. anyways thanks jatin

**Comment by RustoMCSpit** [comment]
just realised theres something a bit similar https://github.com/moddevices/mod-desktop/issues/58

**Comment by RustoMCSpit** [comment]
https://github.com/jatinchowdhury18/RTNeural-NAM is this relevant to this?
